### PR TITLE
Sync with internal PR

### PR DESCRIPTION
**Pull request by jkrup**

### CODE REVIEW

**Review by gvarsanyi — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,10 +1,7 @@
 # SC Gateway
 
 ## Deploy it on Google Cloud Run
-[![Run on Google Cloud](https://deploy.cloud.run/button.svg)](https://deploy.cloud.run?git_repo=https://github.com/Snapchat/pixel-server-gateway.git)
-
-## Deploy it on AWS Cloudformation
-[![](https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png)](https://console.aws.amazon.com/cloudformation/home?#/stacks/create/template?stackName=snap-pixel-gateway&templateURL=https://snap-pixel-server-gateway.s3.us-west-1.amazonaws.com/cloudformation.yaml)
```
Comment (tkuosc) [inline README.md:7]: Is this expected we want to get rid of this? I think we should revert this change?

File: bin/routes/relay.js [hunk 2]
```diff
@@ -89,12 +136,8 @@ class RelayRequest extends incoming_request_1.IncomingRequest {
      */
     hash(value) {
         const sha256 = (0, crypto_1.createHash)('sha256');
-        const hash = sha256.update(value).digest('base64');
```
Comment (tkuosc) [inline bin/routes/relay.js:92]: 🤦

File: README.md (hunk 1 above)
Comment (IBH7493) [reply README.md:7]: Need more information
